techniques/failures/F81.html

```diff
@@ -1,14 +1,17 @@
    </section><section id="examples"><h2>Examples</h2>
       <ul>
-         <li>A user is completing an online form, and the phone number field is required. To indicate that the phone number field is required, the label "Phone Number" is displayed in a color different from the color used for optional fields, without any other indication that "Phone Number" is a required field. A blind or colorblind user may not be able to identify that "Phone Number" is a required field.</li>
+         <li>A user is completing an online form, and the phone number field is required. To indicate that the phone number field is required, the label "Phone Number" is displayed in a color different from the color used for optional fields, without any other indication that "Phone Number" is a required field. A colorblind user may not be able to identify that "Phone Number" is a required field.</li>
          <li>A user submits an online form and leaves a required field blank, resulting in an error. The form field that caused the error is indicated by red text only, without an additional non-color indication that the field caused an error.</li>
       </ul>
       <div class="note">
-         <p>In both examples, the color could be used without failure if the text was sufficiently different in visual presentation (e.g. bold or in a different font) that it would be easily differentiated from the surrounding text if the color were removed. It would also not fail if the color chosen had sufficient luminosity difference (lightness) from the other text that it would be easily be seen as different if viewed in black and white.  In these cases - the information would not be displayed in color (hue) alone but also in "presentation" or "lightness" respectively. </p>
+         <p>In both examples, the color could be used without failure if the text was sufficiently different in visual presentation (e.g. bold or in a different font) that it would be easily differentiated from the surrounding text if the color were removed. It would also not fail if the color chosen had sufficient luminosity difference (lightness) from the other text that it would be easily be seen as different if viewed in black and white. A minimum contrast ratio of 3:1 is considered sufficient. In these cases - the information would not be displayed in color (hue) alone but also in "presentation" or "lightness" respectively. </p>
+      </div>
+      <div class="note">
+         <p>For users of assistive technology, required or error fields must also be perceptible. This is checked with the SCs 1.3.1 and 3.3.1.</p>
```

Comment:
> perceptible

Should that be perceivable (just because that is what WCAG uses elsewhere and it seems to be good to re-use nomenclature :-)